File: DCES_API/src/ru/ipo/dces/clientservercommunication/RegisterToContestRequest.java
package ru.ipo.dces.clientservercommunication;

/**
 * «арегистрироватьс€ дл€ участи€ в контесте
 */
public class RegisterToContestRequest implements Request {
  public String sessionID; //leave it null if this request is anonymous
  public int contestID;
  public UserDescription user;
}

File: Server/Denwer/home/dces-server.ru/www/tests/data/RegisterToContestRequest.php
<?php
require_once('UserDescription.php');
class RegisterToContestRequest{
    public $sessionID;
    public $contestID;
    public $user;

    function __construct(){
        $this->sessionID = null;
        $this->contestID = null;
        $this->user = new UserDescription();
    }
};
?>
